### DISCUSSION TITLE
Remove panels

### DISCUSSION THREAD

**Discussion by maheshnandaniya** [GrapesJS/grapesjs#4268] [Q&A]
Hiii,
I have removed panels using 
`var pn = editor.Panels;
pn.removePanel('views');`

and views panel was removed but how can I remove sidebar 
![IMG_20220419_103933](https://user-images.githubusercontent.com/76687572/163924810-79a0a100-3b23-46f4-8725-93dfa3f29c70.jpg)

Please help me...

**Comment by artf** [comment]
You can use CSS to change the layout of the editor
